src/main/java/seedu/duke/TimeTable.java

```diff
@@ -3,59 +3,70 @@
 import java.time.LocalDate;
 import java.util.ArrayList;
 import java.util.Calendar;
```

Comment:
All of the classes were briefly explained except for the 'main' class. Should the 'main' class also be explained briefly on what it does?
![image](https://user-images.githubusercontent.com/60385689/97513863-45322580-19c8-11eb-8fc4-66d38e255468.png)